### PR TITLE
PS-9053 Release notes for Percona Server for MySQL 8.2.0-1

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
modified:   .github/workflows/main.yml
	new file:   docs/release-notes/8.2.0-1.md
	modified:   docs/release-notes/release-notes-index.md
	modified:   mkdocs-base.yml
	modified:   snippets/storage-engines.md
	modified:   variables.yml

### CODE REVIEW

File: docs/release-notes/8.2.0-1.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+* The insert operations caused `records_in_range` to read too many disk blocks.
+
+* System schema tables with INSTANT ADD columns created before 8.0.29 are incompatible with MySQL versions higher than 8.0.29. Performing DMLs on these tables causes an unexpected server exit.
+
+* A FTS parser plugin now handles single character tokens correctly.
+
+### Deprecation or removal
+
+The following items are deprecated in this release and using any of these items may cause a warning:
```
Comment (patrickbirch) [inline docs/release-notes/8.2.0-1.md]: Maybe

A future release may remove deprecated variables and options. Using these deprecated items may cause a warning. We recommend migrating from deprecated variables and options as soon as possible.

This release deprecates the following variables and options:
			

			* The `binlog_transaction_dependency_tracking` server system variable.
			

			* The `old` and `new` server system variables.
			

			* The `--character-set-client-handshake` server variable.
			

			* `INFORMATION_SCHEMA.PROCESSLIST` and the `SHOW PROCESSLIST` command.
			

			* The `performance_schema_show_processlist` variable.

File: docs/release-notes/8.2.0-1.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+* The `--character-set-client-handshake` server variable.
+
+* `INFORMATION_SCHEMA.PROCESSLIST` and the `SHOW PROCESSLIST` command.
+
+* The `performance_schema_show_processlist` variable.
+
+We recommend migrating from these items as soon as possible. They can be removed in a future release.
+
+The following items are removed in this release and using any of these items causes a warning:
```
Comment (patrickbirch) [inline docs/release-notes/8.2.0-1.md]: If the items are removed, it only causes a warning?

File: docs/release-notes/8.2.0-1.md (hunk 1 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.2.0-1.md]: updated

File: docs/release-notes/8.2.0-1.md (hunk 2 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.2.0-1.md]: updated

File: docs/release-notes/8.2.0-1.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+* The `--character-set-client-handshake` server variable.
+
+* `INFORMATION_SCHEMA.PROCESSLIST` and the `SHOW PROCESSLIST` command.
+
+* The `performance_schema_show_processlist` variable.
+
+This release removes the following variables and options:
+
+* The `WAIT_UNTIL_SQL_THREAD_AFTER_GTIDS()` SQL function. Use `WAIT_FOR_EXECUTED_GTID_SET()` instead.
```
Comment (patrickbirch) [inline docs/release-notes/8.2.0-1.md]: Attempting to invoke this function now causes a syntax error.

File: docs/release-notes/8.2.0-1.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+* The `--abort-slave-event-count` and `--disconnect-slave-event-count` server startup options.
+
+* The `expire_logs_days` server system variable. Use the `binlog_expire_logs_seconds` variable instead.
```
Comment (patrickbirch) [inline docs/release-notes/8.2.0-1.md]: now results in an error.

Comment (alina-derkach-oaza) [reply docs/release-notes/8.2.0-1.md]: added

File: docs/release-notes/8.2.0-1.md (hunk 3 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.2.0-1.md]: added

File: docs/release-notes/8.2.0-1.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
+# Percona Server for MySQL 8.2.0-1 (2024-01-)
+
+Get started with [Quickstart Guide for Percona Server for MySQL].
+
+[Percona Server for MySQL 8.2.0-1] includes all the features and bug fixes available in the
+MySQL 8.2 Community Edition in addition to enterprise-grade features developed by Percona.
+
+## Release highlights
+
+This release merges the MySQL 8.2 code base. In this release Percona does not introduce any new features, improvements or bug fixes.
```
Comment (percona-ysorokin) [inline docs/release-notes/8.2.0-1.md]: We did several visible changes as part of the rebase/merge process
See https://perconadev.atlassian.net/browse/PS-8979 for more details
* Removed remainders of Percona-Specific Encryption features (old 5.7 encrypted bilnog compatibility)
* Removed deprecated "Extended mysqlbinlog" (https://docs.percona.com/percona-server/8.0/extended-mysqlbinlog.html)
* Removed deprecated "rocksdb_strict_collation_check" and "rocksdb_strict_collation_exceptions" RocksDB system variables (https://docs.percona.com/percona-server/8.0/variables.html#rocksdb_strict_collation_check)

**Review by percona-ysorokin — CHANGES_REQUESTED** [review]

File: docs/release-notes/8.2.0-1.md (hunk 5 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.2.0-1.md]: updated

File: docs/release-notes/8.2.0-1.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+Get started with [Quickstart Guide for Percona Server for MySQL].
+
+[Percona Server for MySQL 8.2.0-1] includes all the features and bug fixes available in the
+MySQL 8.2 Community Edition in addition to enterprise-grade features developed by Percona.
+
+## Release highlights
+
+This release merges the MySQL 8.2 code base. Withing this merge Percona introduces several sagnificant changes:
+
+* Removes remainders of Percona-specific encryption features (old 5.7 encrypted bilnog compatibility).
```
Comment (percona-ysorokin) [inline docs/release-notes/8.2.0-1.md]:
```suggestion
* Removes remainders of Percona-specific encryption features (support for custom Percona 5.7 encrypted bilnog format).
```

File: docs/release-notes/8.2.0-1.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+
+* Removes the deprecated `Extended mysqlbinlog` tool.
```
Comment (percona-ysorokin) [inline docs/release-notes/8.2.0-1.md]: @oleksandr-kachan Please review/update this statement.

Comment (percona-ysorokin) [reply docs/release-notes/8.2.0-1.md]: @alina-derkach-oaza just remove this line completely

Comment (alina-derkach-oaza) [reply docs/release-notes/8.2.0-1.md]: fixed

File: docs/release-notes/8.2.0-1.md (hunk 6 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.2.0-1.md]: fixed

**Review by patrickbirch — APPROVED** [review]
LGTM

**Review by percona-ysorokin — APPROVED** [review]
LGTM

**Pull request merged**